chef/cookbooks/network/templates/default/suse-cfg.erb

```diff
@@ -38,8 +38,14 @@ BONDING_MODULE_OPTS=<%=quote("mode=#{@nic.mode} miimon=#{@nic.miimon}")%>
 <% iface["slaves"].each_with_index do |slave,i| -%>
 BONDING_SLAVE_<%=i%>=<%=quote(slave)%>
 <% end -%>
+  <% if iface["mtu"] == 9000 -%>
```

Comment:
Also, is this really only for the magic value of 9000? Or for any value higher than something?